vng_api_common/middleware.py

```diff
@@ -234,31 +234,45 @@ def client_id(self) -> Union[str, None]:
 
         return client_id
 
-    def has_auth(self, scopes: List[str], zaaktype: Union[str, None],
-                 vertrouwelijkheidaanduiding: Union[str, None]) -> bool:
+    def filter_vertrouwelijkheidaanduiding(self, base: QuerySet, value) -> QuerySet:
+        if value is None:
+            return base
 
+        order_provided = VertrouwelijkheidsAanduiding.get_choice(value).order
+        order_case = VertrouwelijkheidsAanduiding.get_order_expression("max_vertrouwelijkheidaanduiding")
+
+        return base.annotate(max_vertr=order_case).filter(max_vertr__gte=order_provided)
+
+    def filter_default(self, base: QuerySet, name, value) -> QuerySet:
+        if value is None:
+            return base
+
+        return base.filter(**{name: value})
+
+    def has_auth(self, scopes: List[str], **kwargs) -> bool:
         if scopes is None:
             return False
 
         scopes_provided = set()
         config = AuthorizationsConfig.get_solo()
+        fields = kwargs or {}
```

Comment:
if no extra `kwargs` are specified, then it's an empty dict by default - maybe change the method `**kwargs` into `**fields` then?

Reply:
fixed